### PR TITLE
Fs 1306 healthcheck

### PR DESCRIPTION
**Pull request by srh-sloan**
Uses the new `/healthcheck` utility in `fsd_utils`. Also:
- Increases memory to 128M after multiple crash loops on deploy to gov paas

Needs updating to latest tag of `fsd_utils` once this PR merged: https://github.com/communitiesuk/funding-service-design-utils/pull/14

### CODE REVIEW

**Comment by gidsg** [comment]
Looks good to me, happy to try and get a version of this into utils.

File: requirements.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.8.tar.gz
+git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1306-healthchecks
```
Comment (gidsg) [inline requirements.in:4]: Change to released version before merging.

**Review by gidsg — APPROVED** [review]
LGTM 👍  but please switch to released version before merging.

File: core/app.py [hunk 2]
```diff
@@ -3,12 +3,16 @@
 """
 import connexion
 from config import Config
-from flask import Flask
+from flask import Blueprint, Flask, current_app
```
Comment (harryyo) [inline core/app.py:6]: Do Blueprint and current_app need to be imported here for the util to work? Is there any way to avoid that if so?

**Review by harryyo — APPROVED** [review]
Great - only one question about imports

File: tests/test_healthcheck.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class TestHealthcheck():
+
+    def testHealthcheckRoute(self, flask_test_client):       
+        expected_dict = {"checks":[{"check_running": "OK"}, {"check_db": "OK"}]}
+
+        response = flask_test_client.get("/healthcheck")
+        assert response.status_code == 200, "Unexpected status code"
+        assert response.json == expected_dict, "Unexpected response body"
```
Comment (Adam-W1) [inline tests/test_healthcheck.py:12]: neat 👍

**Review by Adam-W1 — APPROVED** [review]
lovely 💯

File: core/app.py (hunk 2 above)
Comment (srh-sloan) [reply core/app.py]: Removed :)

**Pull request merged**